quinn-proto/src/endpoint.rs

```diff
@@ -129,7 +121,7 @@ impl Endpoint {
         local_ip: Option<IpAddr>,
         ecn: Option<EcnCodepoint>,
         data: BytesMut,
-    ) -> Option<(ConnectionHandle, DatagramEvent)> {
+    ) -> Option<DatagramEvent> {
```

Comment:
I think this could be factored out into a separate commit? Seems like a somewhat independent change, would make it a little easier to review.

Reply:
Done.